operator/src/main/resources/Operator.properties

```diff
@@ -159,6 +159,24 @@ WLSKO-0201=WL pod shutdown: Failed to shutdown WebLogic server {0} via REST inte
 WLSKO-0202=WL pod shutdown: Failed to shutdown WebLogic server {0} via REST interface due to HTTP request timeout after {1} seconds.
 WLSKO-0203=WL pod shutdown: Failed to shutdown WebLogic server {0} via REST interface due to exception: {1}.
 WLSKO-0204=WL pod shutdown: Retry shutdown of WebLogic server {0} via REST interface.
+WLSKO-0205=WebLogic Domain custom resource conversion webhook, version: {0}, implementation: {1}, build time: {2}
+WLSKO-0206=Could not create webhook liveness file /operator/.webhook_alive
+WLSKO-0207=Unable to read webhook certificate at path {0}
+WLSKO-0208=WebLogic Domain custom resource conversion webhook namespace is: {0}
+WLSKO-0209=The Custom Resource Definition for 'domains.weblogic.oracle' is not installed, waiting for {0} seconds for the CRD to be installed.
+WLSKO-0210=The WebLogic Domain custom resource conversion webhook is shutting down
+WLSKO-0211=Starting Webhook liveness Thread
+WLSKO-0212=WebLogic Domain custom resource conversion webhook identity initialization step failed with exception {0}.
+WLSKO-0213=Domain custom resource converter: InputFile: {0} does not exist or could not be read.
+WLSKO-0214=Domain custom resource converter: The output dir {0} specified by ''-d'' does not exist or could not be read.
+WLSKO-0215=Domain custom resource converter: The output file {0} already exists, remove it and try again. \
+  Use ''-o'' option to overwrite the existing file.
+WLSKO-0216=Domain custom resource converter: Print help message.
+WLSKO-0217=Domain custom resource converter: The directory where the tool will place the converted file. \
+  If not specified, it defaults to the directory of the input file.
+WLSKO-0218=Domain custom resource converter: Name of the converted file. If not specified, it generates the \
+  file by appending "__converted." and the input file extension to the base name of the input file name.
+WLSKO-0219=Domain custom resource converter: Overwrites the existing output file.
```

Comment:
Enable overwriting the existing output file, if any.

Reply:
Also 0216 through 0219 should not have "Domain custom resource converter: " since the string probably looks a little confusing in the generated command line help.

Reply:
Changed in 16f5f6e.